hawkbit-example-mgmt-simulator/src/main/java/org/eclipse/hawkbit/mgmt/client/scenarios/ConfigurableScenario.java

```diff
@@ -45,6 +44,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
+import org.springframework.util.CollectionUtils;
+
+import com.google.common.collect.Lists;
```

Comment:
The usage of `com.google.common.collect.Lists` could be replaced by `final List<MgmtRolloutGroup> result = new ArrayList<>((scenario.getDeviceGroups().size() * 3) + 1);`

Reply:
good catch! Done